frontend/src/components/design-system/typography/BodyText.vue

```diff
@@ -1,33 +1,27 @@
-<script lang="ts">
-import { defineComponent } from 'vue';
-
-import SimpleText from './SimpleText.vue';
-
-export default defineComponent({
-  components: { SimpleText },
-  extends: SimpleText,
-});
-</script>
-
 <script setup lang="ts">
 export interface Props {
+  tag?: string;
```

Comment:
nemel by tag byt enum? nebo alespon nejakej type?

Reply:
a taky bych nedaval ty props jako oprional. Je to mozna vic kodu potom ale je to prehlednejsi pro toho kdo to cte. Takhle se kazdej musi podivat co je default.

Reply:
po prevade ten prvni comment je jenom suggestion ale ten druhej bych fakt nedelal. Optional by meli byt jenom argumenty co nejsou vylozene nutny pro funkcionalitu toho komponentu podle me.

Reply:
OK, upravil jsem. Asi to bude mít vývoj, uvidíme.